### PR TITLE
Manual upgrade: note need for importing tags in EFK upgrade

### PR DESCRIPTION
**Pull request by sosiouxme**
Intended to address:
https://bugzilla.redhat.com/show_bug.cgi?id=1338965#c4

Only needed for OSE 3.2 upgrade notes.

### CODE REVIEW

File: install_config/upgrading/manual_upgrades.adoc [hunk 1]
```diff
@@ -1022,15 +1022,46 @@ endif::openshift-origin[]
...
 template that is created by the deployer:
 +
 ====
 ----
 $ oc process logging-support-template | oc apply -f -
 ----
 ====
+
+. Deployment of logging components is intended to happen automatically
+based on tags being imported into the ImageStreams created in the previous
```
Comment (adellape) [inline install_config/upgrading/manual_upgrades.adoc]: To match this sort of usage elsewhere:

s/ImageStreams/image streams/

File: install_config/upgrading/manual_upgrades.adoc [hunk 2]
```diff
@@ -1022,15 +1022,46 @@ endif::openshift-origin[]
...
+step. However, as not all tags are automatically imported, this mechanism
+has become unreliable as multiple versions are released. Therefore,
+manual importing may be necessary as follows.
++
+For each ImageStream `logging-auth-proxy`, `logging-kibana`,
```
Comment (adellape) [inline install_config/upgrading/manual_upgrades.adoc]: s/ImageStream/image stream/

File: install_config/upgrading/manual_upgrades.adoc [hunk 3]
```diff
@@ -1022,15 +1022,46 @@ endif::openshift-origin[]
...
+$ oc import-image logging-auth-proxy:3.2.1 \
+     --from registry.access.redhat.com/openshift3/logging-auth-proxy:3.2.1
+$ oc import-image logging-kibana:3.2.1 \
+     --from registry.access.redhat.com/openshift3/logging-kibana:3.2.1
+$ oc import-image logging-elasticsearch:3.2.1 \
+     --from registry.access.redhat.com/openshift3/logging-elasticsearch:3.2.1
+$ oc import-image logging-fluentd:3.2.1 \
+     --from registry.access.redhat.com/openshift3/logging-fluentd:3.2.1
+----
+====
```
Comment (adellape) [inline install_config/upgrading/manual_upgrades.adoc]: You can drop the `====` block around these bits. Also the one above that was already in here (1029 - 1033), if you don't mind. These were the only instances of example blocks around code within numbered steps; it gets too busy on the docs.openshift build, and doesn't show up differently anyway on the Portal build.

Just make sure the `+`s are still good for the step after.

**Comment by adellape** [comment]
@sosiouxme Comments inline.

**Comment by sosiouxme** [comment]
@adellape comments addressed

**Comment by adellape** [comment]
@sosiouxme This is published now:

https://docs.openshift.com/enterprise/latest/welcome/revhistory_full.html#thu-jun-30-2016

**Pull request merged**